LenovoLegionToolkit.WPF/Windows/Automation/AutomationPipelineTriggerConfigurationWindow.xaml.cs

```diff
@@ -101,6 +101,7 @@ private void CancelButton_Click(object sender, RoutedEventArgs e)
         IUserInactivityPipelineTrigger ut when ut.InactivityTimeSpan > TimeSpan.Zero => true,
         ITimeAutomationPipelineTrigger => true,
         IWiFiConnectedPipelineTrigger => true,
+        IAutomationPipelineTrigger => true,
```

Comment:
Same here: `IPeriodicAutomationPipelineTrigger` and similar to line 101, you can add a condition:

`IPeriodicAutomationPipelineTrigger papt when papt.PeriodMinutes > 0`